### ISSUE TITLE
Define accessibility and tests for `<map>` and `<layer>` polyfills

### ISSUE THREAD

**Issue by prushforth** [Maps4HTML/Web-Map-Custom-Element#89]
Per this [comment](https://github.com/Maps4HTML/Web-Map-Custom-Element/issues/88#issuecomment-597200561), have created an issue to track what issues we encounter with polyfilling the `<map>` element

**Comment by prushforth** [comment]
I have updated the [demo site](https://geogratis.gc.ca/mapml/en/cbmtile/) so that it will work on Safari, a bit (I don't have routine access to Safari, so hard to test it), by replacing the use of the `<map>` element with a `<mm-app>` element (renamed `<geo-map>` to `<mm-app>`, because it's not geo-specific, and I seriously hope we are able to get the platform to consider `<map>` for I think pretty good reasons, but we should document issues with polyfilling especially for accessibility reasons, as you suggest, under (2).  I take "transparent content model" to mean it presents content that you put there as though the `<map>` element didn't actually exist.  If that is true, the polyfill content should present a root node, perhaps a div as currently implemented, as though it is future <map> element.  I believe that this element should be focusable, so that the user can pan/zoom/query using key strokes appropriately mapped.  The controls inside the shadow root should be reachable using tab stops, and each should be completely actionable using key strokes.

**Comment by Malvoz** [comment]
FYI I filed https://github.com/w3c/html-aam/issues/292.

**Comment by prushforth** [comment]
Re-open if/when necessary.  I would like to not have to discuss the \<map> element more than necessary in future.
